vscode_extension/src/requests/aacRequests.ts

```diff
@@ -0,0 +1,9 @@
+/* tslint:disable */
+/* eslint-disable */
```

Comment:
Why disable the linters, here?

Reply:
I was hunting some linting errors with the generated files, but these don't seem necessary.